website_sale_charge_payment_fee/views/website_sale_view.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<odoo>
+    <template id="assets_frontend" inherit_id="website.assets_frontend">
+      <xpath expr="." position="inside">
+        <script type="text/javascript" src="/website_sale_charge_payment_fee/static/src/js/website_sale_fee.js"/>
+      </xpath>
+    </template>
+
+    <template id="payment_fee" inherit_id="website_sale.payment">
+        <xpath expr="//span[@t-if='acquirer.fees_active']" position="after">
+            <t t-if="acquirer.charge_fee">
+                <span>(Fee:</span>
```

Comment:
why closing the span here? you can nest them

Reply:
done